### PR TITLE
1966 renaming repositories

### LINKED ISSUE #1966: IFileStorageService to Repository
The `IFileStorageService` is really a repository, not a service. Rename it and move it to `repositories/`. Be sure to also move `DirectoryFileStorageService.py`

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do?

Fixes #1966 
Renames repository files to snake case

Add any further explanations here.

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by running unit tests
* [ ] If applicable, add screenshots or log transcripts of the feature working

## Explain Changes

Are the commit messages enough? If not, elaborate.

### CODE REVIEW

File: monkey/monkey_island/cc/repository/file_storage/filesystem_storage.py [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
 from monkey_island.cc.server_utils.file_utils import create_secure_directory
 
-from . import FileRetrievalError, IFileStorageService
+from .i_file_repository import FileRetrievalError, IFileRepository
 
 logger = logging.getLogger(__name__)
 
 
-class DirectoryFileStorageService(IFileStorageService):
+class FilesystemStorage(IFileRepository):
```
Comment (ilija-lazoroski) [inline monkey/monkey_island/cc/repository/file_storage/filesystem_storage.py:14]:
```suggestion
class FileSystemStorage(IFileRepository):
```

**Review by ilija-lazoroski — APPROVED** [review]

File: monkey/tests/unit_tests/monkey_island/cc/resources/test_pba_file_download.py [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
+from monkey_island.cc.repository import FileRetrievalError, IFileRepository
 from monkey_island.cc.resources.pba_file_download import PBAFileDownload
-from monkey_island.cc.services import FileRetrievalError, IFileStorageService
 
 FILE_NAME = "test_file"
 FILE_CONTENTS = b"HelloWorld!"
 
 
-class MockFileStorageService(IFileStorageService):
+class MockFileStorageService(IFileRepository):
```
Comment (mssalvatore) [inline monkey/tests/unit_tests/monkey_island/cc/resources/test_pba_file_download.py:15]:
```suggestion
class MockFileRepository(IFileRepository):
```

File: monkey/tests/unit_tests/monkey_island/cc/resources/test_pba_file_upload.py [hunk 3]
```diff
@@ -67,7 +67,7 @@ def file_storage_service():
 @pytest.fixture
 def flask_client(build_flask_client, file_storage_service):
     container = StubDIContainer()
-    container.register_instance(IFileStorageService, file_storage_service)
+    container.register_instance(IFileRepository, file_storage_service)
```
Comment (mssalvatore) [inline monkey/tests/unit_tests/monkey_island/cc/resources/test_pba_file_upload.py:70]:
```suggestion
    container.register_instance(IFileRepository, file_repository)
```

**Review by mssalvatore — CHANGES_REQUESTED** [review]
Double check the naming of everything in the tests. There are classes, fixtures, and local variables named `file_storage_service` or similar.

File: monkey/monkey_island/cc/repository/file_storage/file_system_storage.py [hunk 4]
```diff
@@ -6,12 +6,12 @@
...
 from monkey_island.cc.server_utils.file_utils import create_secure_directory
 
-from . import FileRetrievalError, IFileStorageService
+from .i_file_repository import FileRetrievalError, IFileRepository
 
 logger = logging.getLogger(__name__)
 
 
-class DirectoryFileStorageService(IFileStorageService):
+class FileSystemStorage(IFileRepository):
```
[the remark this answers is not in the document]
Comment (VakarisZ) [reply monkey/monkey_island/cc/repository/file_storage/file_system_storage.py:14]: Why? Is this a general guideline that if we have an interface `IAnimal`, then concrete class will be called `CatAnimal(IAnimal)`?

**Review by mssalvatore — APPROVED** [review]

**Pull request merged**